browse/static/js/huggingface.js

```diff
@@ -0,0 +1,326 @@
+// Labs integration for displaying machine learning demos from huggingface.co
+
+(function () {
+    const container = document.getElementById("huggingface-output");
+    const containerAlreadyHasContent = container.innerHTML.trim().length > 0;
+
+    // This script is invoked every time the Labs toggle is toggled, even when
+    // it's toggled to disabled. So this check short-circuits the script if the
+    // container already has content.
+    if (containerAlreadyHasContent) {
+        container.innerHTML = "";
+        container.setAttribute("style", "display:none");
+        return;
+    } else {
+        container.setAttribute("style", "display:block");
+    }
+
+    // Get the arXiv paper ID from the URL, e.g. "2103.17249"
+    // (this can be overridden for testing by passing a override_paper_id query parameter in the URL)
+    const params = new URLSearchParams(document.location.search);
+    const arxivPaperId = params.get("override_paper_id") || window.location.pathname.split('/').reverse()[0];
+    if (!arxivPaperId) return;
+
+    const huggingfaceApiHost = "https://huggingface.co/api";
+    const huggingfaceRepo = `${huggingfaceApiHost}/arxiv/${arxivPaperId}/repos`;
+    const huggingfacePaperApi = `${huggingfaceApiHost}/papers/${arxivPaperId}?field=comments`;
```

Comment:
Are comments needed?

```suggestion
    const huggingfacePaperApi = `${huggingfaceApiHost}/papers/${arxivPaperId}`;
```